### PR TITLE
update SnoopCompile

### PR DESCRIPTION
**Pull request by aviatesk**

### CODE REVIEW

**Comment by aminya** [comment]
Please merge https://github.com/JunoLab/Atom.jl/pull/251. I fixed the tests and depcrecated macros

File: .github/workflows/SnoopCompile.yml [hunk 1]
```diff
@@ -3,13 +3,12 @@ name: SnoopCompile
...
   - push
 
-
 jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['nightly']
+        julia-version: ['1.3']
```
Comment (aminya) [inline .github/workflows/SnoopCompile.yml]: You should use `1.4.0-rc1` to get the latest SnoopCompile feature.

Comment (aviatesk) [reply .github/workflows/SnoopCompile.yml]: Is the cutting edge feature important ?
If not, I would prefer generating against the stable release.

Comment (aminya) [reply .github/workflows/SnoopCompile.yml]: Here it is different. A `1.4` feature is used to remove invalid precompilation sentences. So using older than `1.4` is less stable in this case:
https://github.com/timholy/SnoopCompile.jl/blob/8d73d1f1d70ca2576e3ef3f6f1d0b80080a4973e/src/parcel_snoopi.jl#L276

Comment (aviatesk) [reply .github/workflows/SnoopCompile.yml]: I see. Thanks !

**Pull request merged**